### ISSUE TITLE
Suggestion: add "Load list" and "Save list" items in File menu or as direct buttons in the menu bar

### ISSUE THREAD

**Issue by pmascalchi** [DrHenrikPersson/macro_manager#1]
This would just save lists as "macroManagerList_XXX.txt" files...
The default list would be superseded but you could still load it back or close/open again the plugin...

**Comment by DrHenrikPersson** [comment]
You mean like a backup of the current settings file where all the macro paths are stored? That would make sense, letting the user have one set of macros for each of their analysis workflows.

**Comment by pmascalchi** [comment]
Exactly, to adjust to different projects. You can even think about Core Facilities, they could define lists depending on users.
